amps/ags/rm-community/rm-community-repo/source/java/org/alfresco/module/org_alfresco_module_rm/disposition/DispositionServiceImpl.java

```diff
@@ -1113,7 +1109,7 @@ public Void doWork()
                     }
 
                     List<DispositionActionDefinition> dispositionActionDefinitions = dispositionSchedule.getDispositionActionDefinitions();
-                    DispositionActionDefinition currentDispositionActionDefinition = null;
+                    DispositionActionDefinition currentDispositionActionDefinition;
                     DispositionActionDefinition nextDispositionActionDefinition = null;
```

Comment:
Remove assigning null to nextDispositionActionDefinition, can maintain without it.

Reply:
No, we can't remove this initialization.